<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arc</class>
 <widget class="QMainWindow" name="arc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Monospace</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>arc</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>350</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Main</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="horizontalWidget">
       <attribute name="title">
        <string>Move</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="verticalWidget">
           <attribute name="title">
            <string>Straight</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QToolBox" name="toolBox">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="verticalWidget">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>282</width>
                 <height>193</height>
                </rect>
               </property>
               <attribute name="label">
                <string>X-Y-Offset (mm)</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QWidget" name="horizontalWidget" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_3">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="maximumSize">
                      <size>
                       <width>25</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>X:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="pat_str_offset_x_doubleSpinBox">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>50</height>
                      </size>
                     </property>
                     <property name="minimum">
                      <double>-10000000000.000000000000000</double>
                     </property>
                     <property name="maximum">
                      <double>10000000000.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>1.000000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="horizontalWidget_2" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="maximumSize">
                      <size>
                       <width>25</width>
                       <height>16777215</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>Y:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QDoubleSpinBox" name="pat_str_offset_y_doubleSpinBox">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>50</height>
                      </size>
                     </property>
                     <property name="minimum">
                      <double>-10000000000.000000000000000</double>
                     </property>
                     <property name="maximum">
                      <double>10000000000.000000000000000</double>
                     </property>
                     <property name="singleStep">
                      <double>1.000000000000000</double>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pat_str_offset_pushButton">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Add</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="verticalWidget">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>282</width>
                 <height>193</height>
                </rect>
               </property>
               <attribute name="label">
                <string>Angle-Length</string>
               </attribute>
               <layout class="QVBoxLayout" name="verticalLayout_5"/>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="verticalWidget">
           <attribute name="title">
            <string>Arc</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
          <widget class="QWidget" name="verticalWidget">
           <attribute name="title">
            <string>Beziere</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3"/>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="verticalGroupBox">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListWidget" name="pat_actions_listWidget"/>
           </item>
           <item>
            <widget class="QWidget" name="horizontalWidget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QPushButton" name="pat_actions_up_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>UP</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pat_actions_down_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>DOWN</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pat_actions_remove_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>REMOVE</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pat_actions_go_pushButton">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="text">
                 <string>GO</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="verticalGroupBox_2">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGraphicsView" name="pat_actions_preview_graphicsView"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>Store-Pattern</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Pattern</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>G-Code</string>
       </attribute>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <row>
       <property name="text">
        <string>#00001</string>
       </property>
       <property name="background">
        <color>
         <red>243</red>
         <green>243</green>
         <blue>243</blue>
        </color>
       </property>
      </row>
      <row>
       <property name="text">
        <string>#00002</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RSV</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CMD</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RCV</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>RTN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>23:59:59</string>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>XYZ</string>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>G00;X10000;Y-250000;</string>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>OK</string>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>X:15000;Y:-5000;</string>
       </property>
      </item>
      <item row="0" column="5">
       <property name="text">
        <string>OK</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>00:00:10</string>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>MIL</string>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>S15000;</string>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>OK</string>
       </property>
      </item>
      <item row="1" column="4">
       <property name="text">
        <string/>
       </property>
      </item>
      <item row="1" column="5">
       <property name="text">
        <string>IDL</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
